.travis.yml

```diff
@@ -7,6 +7,16 @@ branches:
   only:
   - master
 
+notifications:
+  webhooks:
+    urls:
+      - secure: "ekgTSMPPLMC8Yz2zCg8Tbh8nFm/Soj5jX1YjbZQ1ZaZZk35YcBaqRclU0yEM1YnOSSBWP7CefbwazKPqjlBBKa6bou2Nz1TUjeZxrE9yeczY+50QF7Kb4WnE4uDAQYTTi8x6WPLIlLyCHiwVcKOzRGC6MxSpcGXf57YamE5fatgfsUXGxqcUVADk/d2kh9a+v1y/lwIriZqWIEQ585MAZ34tWpz+2G8q97i76WDSvQ4bm6KAWu+ZFvGM9EtFRrPI6iHCLGnHeo+icUX0aIlsaN3SDoYhrkkRbHgyL3Z3ZYZwXa0ewKyiKk6GijFPJhAqGbCiH0B/4j/zWwYsb25SMLmaaSwHmqVHevyYHbr/lRaiamP8K6ERt37GQ/qp42o6xN5uZGf141uZ6FWpWkRNlmECi9pMAPeYGE6CNL19npoAaZqy4LutVVmiSfdYNFZFwIErDR3k7XB/M5Wmi/h0UIAgDJk4v+Hd7StS+nMCQKzOn2zqq/pEDT7uX6b03QORgDjT/RJHbjaB72DqzrGa9DpJNALOAxgf0Oeqyin6EqHDadumDZmDYkrohYAqtP7GVirKFZE1ZJla6pn50j5HuKRkPqnkOtg2yd480pqsyOr2eu1FUD8oku7U5ZntdZNeuZIS3qWS7jARdrYoIm3VKq/mN50o+XQgIizD98wj4kQ="
```

Comment:
We cannot do this as an env variable, because the env variables are not available when the notifications are send.
But you can encrypt the url - but you cannot have linebreaks.

Reply:
> though you cannot have linebreaks

that's fine, but the following should work, or doesn't it?

```yaml
notifications:
  webhooks:
    urls:
      - secure: >- 
        ekgTSMPPLMC8Yz2zCg8Tbh8nFm/Soj5jX1YjbZQ1ZaZZk35YcBaqRclU0yEM1YnOSSBWP7CefbwazKPqjlBBKa6bou2Nz1TUjeZxr
        E9yeczY+50QF7Kb4WnE4uDAQYTTi8x6WPLIlLyCHiwVcKOzRGC6MxSpcGXf57YamE5fatgfsUXGxqcUVADk/d2kh9a+v1y/lwIriZ
        qWIEQ585MAZ34tWpz+2G8q97i76WDSvQ4bm6KAWu+ZFvGM9EtFRrPI6iHCLGnHeo+icUX0aIlsaN3SDoYhrkkRbHgyL3Z3ZYZwXa0
        ewKyiKk6GijFPJhAqGbCiH0B/4j/zWwYsb25SMLmaaSwHmqVHevyYHbr/lRaiamP8K6ERt37GQ/qp42o6xN5uZGf141uZ6FWpWkRN
        lmECi9pMAPeYGE6CNL19npoAaZqy4LutVVmiSfdYNFZFwIErDR3k7XB/M5Wmi/h0UIAgDJk4v+Hd7StS+nMCQKzOn2zqq/pEDT7uX
        6b03QORgDjT/RJHbjaB72DqzrGa9DpJNALOAxgf0Oeqyin6EqHDadumDZmDYkrohYAqtP7GVirKFZE1ZJla6pn50j5HuKRkPqnkOt
        g2yd480pqsyOr2eu1FUD8oku7U5ZntdZNeuZIS3qWS7jARdrYoIm3VKq/mN50o+XQgIizD98wj4kQ=
```

(https://stackoverflow.com/questions/3790454/how-do-i-break-a-string-over-multiple-lines)

> But you can encrypt the url

How is the url encrypted? Can you give a small comment saying `This notifies us on gitter.im `

Reply:
Did you use https://docs.travis-ci.com/user/environment-variables/#encrypting-environment-variables ?

Reply:
Yes, kinda `travis encrypt SOME_URL -r seqan/seqan3`, the `-r` part is the repo you want this to encrypt for